adafruit_epd/il0373.py

```diff
@@ -112,21 +130,21 @@ def power_up(self):
         self.command(_IL0373_VCM_DC_SETTING, bytearray([0x0A]))
         time.sleep(0.05)
 
-    def power_down(self):
+    def power_down(self) -> None:
         """Power down the display - required when not actively displaying!"""
         self.command(_IL0373_CDI, bytearray([0x17]))
         self.command(_IL0373_VCM_DC_SETTING, bytearray([0x00]))
         self.command(_IL0373_POWER_OFF)
 
-    def update(self):
+    def update(self) -> None:
         """Update the display from internal memory"""
         self.command(_IL0373_DISPLAY_REFRESH)
         time.sleep(0.1)
         self.busy_wait()
         if not self._busy:
             time.sleep(15)  # wait 15 seconds
 
-    def write_ram(self, index):
+    def write_ram(self, index: Union[0, 1]) -> Any:
```

Comment:
This is returning the result of `self.command()` which is typed as `int` so I think the return type here can be `int` instead of `Any`

This would be the case for all of the `write_ram` functions in the other specific display driver files as well.